kitchen/forms.py

```diff
@@ -0,0 +1,86 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from kitchen.models import Dish, Cook
+
+
+class DishForm(forms.ModelForm):
+    cooks = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+    )
+
+    class Meta:
+        model = Dish
+        fields = "__all__"
+
+
+class DishSearchForm(forms.Form):
+    name = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": "Search by dish name",
+            }
+        )
+    )
+
+
+class CookCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Cook
+        fields = UserCreationForm.Meta.fields + (
+            "years_of_experience",
+            "first_name",
+            "last_name",
+        )
+
+
+class CookYearsUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Cook
+        fields = ["years_of_experience"]
+
+    def clean_years_of_experience(self):
+        return validate_years_of_experience(self.cleaned_data["years_of_experience"])
+
+
+def validate_years_of_experience(
+    years_of_experience,
+):
+    if years_of_experience < 0:
+        raise ValidationError("Should be more than 0")
+    elif not years_of_experience.isdigit():
```

Comment:
Wydaje mi się, że najpierw powinnismy sprawdzić, czy to liczba, a dopiero potem czy jest mniejsza od zera